assessment.sql

```diff
@@ -6,23 +6,46 @@ CREATE DATABASE "sql-crud-assessment";
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
 -- your code here
+CREATE TABLE licensees(
+id SERIAL PRIMARY KEY,
+given_name CHARACTER VARYING,
+gender CHARACTER VARYING (1),
+height INTEGER,
+weight INTEGER
```

Comment:
Commas separate column definitions; the comma that is missing at the end of this line will prevent this statement from working properly.